packages/store/src/System.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+// TODO: Use existing ERC2771 implementations
```

Comment:
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/metatx/ERC2771Context.sol

Reply:
yeah true, will remove the comment. For future reference - we don't need a trusted forwarder check here, because systems don't contain own storage, but instead act on the caller's storage (either via delegatecall or via callbacks, see `StoreSwitch`), so any caller can be "trusted".